[2024-09-03T19:25:38.301+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: SitesWebScraping.create_container_kabum manual__2024-09-03T19:25:36.476987+00:00 [queued]>
[2024-09-03T19:25:38.308+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: SitesWebScraping.create_container_kabum manual__2024-09-03T19:25:36.476987+00:00 [queued]>
[2024-09-03T19:25:38.309+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 1
[2024-09-03T19:25:38.324+0000] {taskinstance.py:2192} INFO - Executing <Task(CreateContainerInDataLake): create_container_kabum> on 2024-09-03 19:25:36.476987+00:00
[2024-09-03T19:25:38.328+0000] {standard_task_runner.py:60} INFO - Started process 89 to run task
[2024-09-03T19:25:38.332+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'SitesWebScraping', 'create_container_kabum', 'manual__2024-09-03T19:25:36.476987+00:00', '--job-id', '321', '--raw', '--subdir', 'DAGS_FOLDER/DagWebScraping.py', '--cfg-path', '/tmp/tmp9yuf0tcy']
[2024-09-03T19:25:38.336+0000] {standard_task_runner.py:88} INFO - Job 321: Subtask create_container_kabum
[2024-09-03T19:25:38.380+0000] {task_command.py:423} INFO - Running <TaskInstance: SitesWebScraping.create_container_kabum manual__2024-09-03T19:25:36.476987+00:00 [running]> on host 0648ceca68fa
[2024-09-03T19:25:38.456+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='SitesWebScraping' AIRFLOW_CTX_TASK_ID='create_container_kabum' AIRFLOW_CTX_EXECUTION_DATE='2024-09-03T19:25:36.476987+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-03T19:25:36.476987+00:00'
[2024-09-03T19:25:38.459+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-09-03T19:25:38.460+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'mkdir -p /usr/local/***/datalake/kabum=2024-09-03']
[2024-09-03T19:25:38.469+0000] {subprocess.py:86} INFO - Output:
[2024-09-03T19:25:38.472+0000] {subprocess.py:93} INFO - mkdir: cannot create directory ‘/usr/local/***/datalake’: Permission denied
[2024-09-03T19:25:38.473+0000] {subprocess.py:97} INFO - Command exited with return code 1
[2024-09-03T19:25:38.484+0000] {taskinstance.py:2699} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 433, in _execute_task
    result = execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/bash.py", line 212, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2024-09-03T19:25:38.487+0000] {taskinstance.py:1138} INFO - Marking task as FAILED. dag_id=SitesWebScraping, task_id=create_container_kabum, execution_date=20240903T192536, start_date=20240903T192538, end_date=20240903T192538
[2024-09-03T19:25:38.501+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 321 for task create_container_kabum (Bash command failed. The command returned a non-zero exit code 1.; 89)
[2024-09-03T19:25:38.543+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-09-03T19:25:38.558+0000] {taskinstance.py:3281} INFO - 0 downstream tasks scheduled from follow-on schedule check
